content/en/docs/tutorials/online-training/overview.md

```diff
@@ -15,10 +15,14 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Amazon EKS Deep Dive with Hands-On Labs (Linux Academy)] (https://linuxacademy.com/amazon-web-services/training/course/name/amazon-eks-deep-dive)
 
+* [Online Kubernetes Training with CKA & CKAD prep (Component Soft)](https://www.componentsoft.io/kubernetes-administration-training-2/)
```

Comment:
The existing list is sorted alphabetically. Add new entries sorted into the correct location.

Reply:
Sorting fixed